app/uk/gov/hmrc/customs/declarations/information/services/responseFilterServices.scala

```diff
@@ -26,8 +26,8 @@ import scala.xml.{NodeSeq, Text, TopScope}
 class FullResponseFilterService @Inject()() extends ResponseFilterService {
 
   override protected val NameSpaceP="http://gov.uk/customs/FullDeclarationDataRetrievalService"
-  override protected val rootElementLabel: String = ""
-  override protected val detailsElementLabel: String = ""
+  override protected val rootElementLabel: String = "DeclarationFullResponse"
```

Comment:
these are both deliberately empty - they're not used. Maybe need to add a comment indicating that.